lib/redmine_required_field/patches/issues_controller_patch.rb

```diff
@@ -36,6 +36,30 @@ def update
 
 
       def rrf_params_match_settings(rrf_setting)
+        #check role and assignee behavior(negate match for given roles if checkbox is set, and only apply filter if attached to only is set and user matches attached to )
+        user_is_assigned_to = User.current.id==@issue.assigned_to_id
```

Comment:
please leabe space before and after '=='

Reply:
updated on next commit